server/jdkhttp-server/src/main/scala/sttp/tapir/server/jdkhttp/internal/JdkHttpRequestBody.scala

```diff
@@ -29,10 +36,42 @@ private[jdkhttp] class JdkHttpRequestBody(createFile: ServerRequest => TapirFile
         val file = createFile(serverRequest)
         Files.copy(asInputStream, file.toPath, StandardCopyOption.REPLACE_EXISTING)
         RawValue(FileRange(file), Seq(FileRange(file)))
-      case _: RawBodyType.MultipartBody => throw new UnsupportedOperationException("MultipartBody is not supported")
+      case m: RawBodyType.MultipartBody => RawValue.fromParts(multiPartRequestToRawBody(serverRequest, m))
     }
   }
 
+  private val boundaryPrefix = "boundary="
+  private def extractBoundary(request: HttpExchange): String = {
+    Option(request.getRequestHeaders.getFirst("Content-Type"))
+      .flatMap(
+        _.split(";")
+          .find(_.trim().startsWith(boundaryPrefix))
+          .map(line => s"--${line.trim().substring(boundaryPrefix.length)}")
+      )
+      .getOrElse(throw new IllegalArgumentException("Unable to extract multipart boundary from multipart request"))
```

Comment:
I assumed exceptions would be the way to do error hendling here because of the `UnsupportedOperationException` and the returntype didn't seem to offer way to return an error. Am i correct?

Reply:
Hm not sure about logging, but probably supplying a default slf4j logging interceptor would be a good idea, maybe a separate task :)

Reply:
Ignore that, there *is* default logging using java's logging